spec/helpers/menu_heplper_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe MenuHelper, type: :helper do
+  let!(:menu_1) { create(:menu, menu_name: "menu 1")}
+  let!(:menu_2) { create(:menu, menu_name: "menu 2")}
+  let!(:menu_3) { create(:menu, menu_name: "menu 3")}
+  let!(:menu_4) { create(:menu, menu_name: "menu 4")}
+
+  describe '#menu_list' do
+    specify do
+      expect(helper.menu_list).to eq [["menu 1", 1], ["menu 2", 2], ["menu 3", 3], ["menu 4", 4]]
```

Comment:
`expect(helper.menu_list).to eq [["menu 1", menu_1], ["menu 2", menu_2.id], ["menu 3", menu_3.id], ["menu 4", menu_4.id]]`